frontend/src/metabase/common/hooks/entity-framework/use-entity-query/use-entity-query.ts

```diff
@@ -59,7 +60,10 @@ export const useEntityQuery = <TId, TItem, TQuery = never>(
     requestType,
   }: UseEntityOwnProps<TId, TItem>,
 ): UseEntityQueryResult<TItem> => {
-  const options = { entityId, requestType };
+  const options = useMemo(
+    () => ({ entityId, requestType }),
+    [entityId, requestType],
+  );
```

Comment:
this just suppresses a warning that rtk was making when passing these options to the selector (the object is different on every rerender)